src/extras/core/Path.js

```diff
@@ -151,7 +151,8 @@ THREE.Path.prototype = Object.assign( Object.create( THREE.CurvePath.prototype )
 			xRadius, yRadius,
 			aStartAngle, aEndAngle,
 			aClockwise,
-			aRotation || 0 // aRotation is optional.
+			aRotation || 0, // aRotation is optional.
+			this.curves.length
```

Comment:
Here I don’t worry about performance (I did for `deltaAngle` and made a mistake), I worry about code sanity. I don’t understand why `Path.getPoints` doesn’t use the `curves` array to get the points for each part of the curve (expect for `lineTo` because there is no calculation in that case) from an instance of the class dedicated to manage each kind of curve.
If you look at the `splineThru` case, a new instance of `THREE.SplineCurve` is created just to be able to call its `getPointAt`. In the other cases, including ellipse before my modification, a local calculation is made in `Path.getPoints` with the parameters stored in `actions`. Why `Path.getPoints` never used the dedicated class instances that are stored in `curves`?
As I didn’t want to change all `Path.getPoints` but only to the ellipse part, I add the index in actions.
If that is not correct, there are two solutions.
The first one is the ugly trick used for `splineThru`, create a new instance of `EllipseCurve` using parameters stored in `actions` and use this instance instead of the one stored in `curves`.
The second one (my preference), add another index dedicated to the `curves` array in the for loop of `Path.getPoints` and increment it in each case where something have been pushed in `curves` array (all cases except 'moveTo'). Now each case can ask the dedicated instance stored in curves to do the calculation. But this solution will impact all the cases.
The problem I initialy pointed is mainly due to the fact that `Path.getPoints` and `CurvePath.getPoint` don’t use the same way to do the job. `CurvePath.getPoint` uses the `curves` array, `Path.getPoints` doesn’t.